### DISCUSSION TITLE
Cannot import OSC

### DISCUSSION THREAD

**Discussion by zappfinger** [pyodide/pyodide#2524] [General]
I try to import OSC but it fails.
Since pyodide is now part of OSC, I would expect this to work...

**Comment by ryanking13** [comment]
Do you mean `python-osc`?

Since it is a pure python package, you can install it with `micropip`, though it won't be very useful as Pyodide does not support socket connection.

```js
await pyodide.loadPackage("micropip")
pyodide.runPython(`
  import micropip
  await micropip.install("python-osc")
  import pythonosc
`
```
